charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/job-patch/job-patchWebhook.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - --webhook-name={{ template "kube-prometheus-stack.fullname" . }}-admission
             - --namespace={{ template "kube-prometheus-stack.namespace" . }}
             - --secret-name={{ template "kube-prometheus-stack.fullname" . }}-admission
-            - --patch-failure-policy={{ .Values.prometheusOperator.admissionWebhooks.failurePolicy }}
```

Comment:
I ask myself, if it make sense to have `Fail` here if `.Values.prometheusOperator.admissionWebhooks.failurePolicy` is `IgnoreOnInstallOnly`

Because after the initial install, the policy can be set to Failure and the helm hook always runs after a complete installation.

What did you thing? @DrBu7cher

Reply:
What would be your suggestion?

Reply:
The whole idea behind this PR is that the value for this policy should be „Ignore“ when installing and „Fail“ when not installing. Previously, the hook would have been set up with the flag set to a value that is invalid, as the error message in my opening comment showed.

What value or behavior would make more sense to you?

Reply:
> The whole idea behind this PR is that the value for this policy should be „Ignore“ when installing and „Fail“ when not installing.

I'm aware of this, since I [implement](https://github.com/prometheus-community/helm-charts/pull/3066) the idea of IgnoreOnInstallOnly, but I'm using cert-manager solution and I was not aware that the patch job also patch the policy.

After review the PR, I thought since the patch job runs after installation as additional hook, it cloud already set the failure-policy to `Failure`, if `.Values.prometheusOperator.admissionWebhooks.failurePolicy` is set to `IgnoreOnInstallOnly`. It would omit the required of a second `helm upgrade` after the initial deployment.

The current state in the PR is also fine for me. Its more like an optimization, if it make sense.

Reply:
Otherwise, if the value here is always equal to the value inside webhooks, I does it make sense to have this parameter? If omit, the patch pod would not touch failurePolicy. That also seems fine to me, since we maintain the value directly via helm.